src/spark.cpp

```diff
@@ -0,0 +1,9 @@
+#include "spark.h"
+
+namespace spark {
+
+bool GetOutPoint(COutPoint& outPoint, const spark::Coin coin) {
```

Comment:
This function is called in a hot loop in `CWallet::GetAvailableSparkCoins`. Equivalent Lelantus functions perform a disk read here.

Reply:
I see your point, actually for Lelantus much time takes to get mints from db, reading block from disk is very fast compared to reading data from db, I will suggest to keep this for now, and try to improve during testnet testing.

Reply:
For spark it is not looking into db every time, we keep mints in memory, and only touch db during adding or removing coins.